### ISSUE TITLE
How do I add a custom domain to a page?

### ISSUE THREAD

**Issue by o-bardiuk** [cloudflare/cloudflare-docs#11646]
### Which Cloudflare product(s) does this pertain to?

Pages

### Subject Matter

How to add a custom domain to a page

### Content Location

https://developers.cloudflare.com/pages/

### Additional information

I can't find any info on this matter. It should be possible though, right??

**Comment by haleycode** [comment]
Hi @o-bardiuk! I think this answers your question: https://developers.cloudflare.com/pages/platform/custom-domains/

If not, please feel free to let us know if there are any specific updates or changes to the docs that you believe would be useful.

**Comment by o-bardiuk** [comment]
My issue was that you don't see any options about custom domain if you don't add any to your account. It seems like all custom domain options and menu entries are invisible unless you create a custom domain. It's a little bit unclear and not user-friendly, I'd say.

UPD: I did see the https://developers.cloudflare.com/pages page but I haven't seen any "domain" section and I definitely didn't see that it is in the Platform section in the very end. I have no idea what is the logic behind putting it there. Also Google didn't show me this page. Now that I visited it, it also appears in the search. But that wasn't the case before.

I guess the link to the "custom domain" sub-section must be somewhere on the main page of the "Pages" chapter, but not hidden in the platform section.

@haleycode
